docs: Stacker instruction manual

jwwojak opened the pull request:
Committing Stacker instruction manual files.

# Overview

This is the Stacker instruction manual.

## Test Plan and Hands on Testing

Will review the text for the usual style, grammar, and the usual writing stuff. Also, check to make sure links work and images appear as expected. This doesn't have some of the recent CSS, but posting it up anyway to get started.

## Changelog

No changes. This is new content.

## Review requests

Nothing specific. Please review and comment.

## Risk assessment

Low to medium as this is a new docs feature. Who knows what horrors may lurk within.

jwwojak committed:
Stacker manual first commit

Committing Stacker instruction manual files.

jwwojak committed:
Stacker manual first commit

Committing Stacker instruction manual files.

jwwojak committed:
Stacker manual first commit

Committing Stacker instruction manual files.

ecormany commented on docs/stacker-manual/docs/index.md:
We should come up with a standard format for cover pages. Probably one of:
- Same as print covers. Just logo, title, hero, company name, date. 
- Print cover info + Product Description. This is probably good for SEO or RAG or whatever.

Either way, I would move Manufacturer Description to the Additional Product Information page. Some print pubs have that ordering anyway.

jwwojak replied:
Sure, will move the mfg description to the back, to the Additional Info page.

Here's my thinking about the home/landing page: I put the product description on this page because without it, the page doesn't have much information value. It's just an image and date of publication. The product summary seems nice here as it adds a small amount of information to what would otherwise be just a page with a picture. A little different than the manuals, but as a landing page, now it shows you the thing and provides a short description.

ecormany commented on docs/stacker-manual/docs/index.md:
```diff
@@ -0,0 +1,30 @@
+  <img src="/images/tm-opentrons-flex.png" alt="flex-tm" style="width:70%">
+</a>
+
+# **Stacker Module GEN1 Instruction Manual**
```
H1 should never need or have bold styling.

jwwojak replied:
Removed bold, but the text looks too skinny to me.

ecormany replied:
We'll make sure we have the right weight across the board in our theme CSS file.

ecormany commented on docs/stacker-manual/docs/index.md:
```diff
@@ -0,0 +1,30 @@
+<b>[Opentrons Labworks Inc](https://opentrons.com/)</b><br>
+August 2025
+///
+</figure markdown>
```
Closing HTML tags don't have attributes. (This would be an error if you were doing HTML validation.)

jwwojak replied:
My code here is super-wonky. Going to fix it.

jwwojak replied:
Refactored code. Much simpler here:

```
![Stacker hero image](images/stacker-hero.png)

<div style="text-align: center;">
    <strong>Opentrons Labworks Inc</strong><br>August 2025
</div>
```

ecormany commented on docs/stacker-manual/docs/index.md:
```diff
@@ -0,0 +1,30 @@
+
+## Product Description
+
+The Opentrons Flex&trade; Stacker is an external module that provides automated, high-capacity storage for ANSI/SLAS compatible labware such as well plates, tip racks, and reservoirs. It also includes a shuttle, which moves labware from the Stacker to the deck where it can be accessed manually or by the Flex Gripper. While ideal for high-throughput NGS workflows, the Stacker can help streamline a ...
```
Let's use unicode everywhere. It makes the source more readable. So `™` here.

jwwojak replied:
Unicode...Unicode everywhere.

![buzz](https://github.com/user-attachments/assets/5d382bce-d432-4f20-9849-27be3d5ec6ac)

jwwojak replied:
Using unicode everywhere.

ecormany commented on docs/stacker-manual/docs/stacker-compliance.md:
I would recommend not repeating "stacker" in file names. The ultimate URL for this page would be something like `docs.opentrons.com/stacker-manual/stacker-compliance`. This is one reason I think just `stacker` would be better for the parent directory as well: `docs.opentrons.com/stacker/compliance` is very clean and includes all the same information.

jwwojak replied:
Shortening the file names. Dropping `stacker-`.

ecormany commented on docs/stacker-manual/docs/stacker-compliance.md:
```diff
@@ -0,0 +1,162 @@
+
+- 100–240 VAC, 50/60 Hz
+- 6.4 A/115 VAC, 3.2 A/230 VAC
+- Mains fluctuation: &plusmn; 10%
```
Another place for Unicode. ±

ecormany commented on docs/stacker-manual/docs/stacker-compliance.md:
```diff
@@ -0,0 +1,162 @@
+    Changes or modifications to this module not expressly approved by Opentrons could void the user’s authority to operate the equipment. This device complies with part 15 of the FCC Rules.
+
+!!!note
+    This equipment has been tested and found to comply with the limits for a Class A digital device, pursuant to part 15 of the FCC rules. These limits are designed to provide a reasonable protection against harmful interference when the equipment is operated in a commercial environment. This equipment generates, uses, and can radiate radio frequency energy and, if not installed and used in accor ...
```
This is the weird exception where "Warning:" and "Note:" should be in plain text. That's part of the body of the FCC boilerplate. We, Opentrons, are not calling these out as warnings or notes according to our criteria.

jwwojak replied:
Removed call-out text. Instead, the FCC warning and note will be H4s under the FCC H3.

They won't appear in the page-level TOC as `toc-depth: 3`, but the FCC section will.

ecormany commented on docs/stacker-manual/docs/stacker-compliance.md:
```diff
@@ -0,0 +1,162 @@
+
+Le présent produit est conforme aux spécifications techniques applicables d’Innovation, Sciences et Développement économique Canada.
+
+CISPR 11 Class A
```
H4
```suggestion
#### CISPR 11 Class A
```

jwwojak replied:
Make H4

ecormany commented on docs/stacker-manual/docs/stacker-compliance.md:
```diff
@@ -0,0 +1,162 @@
+
+CISPR 11 Class A
+
+!!!caution
```
Again, not our admonition.

jwwojak replied:
Removed admonition in favor of **Caution** as part of the text.

ecormany commented on docs/stacker-manual/docs/stacker-compliance.md:
```diff
@@ -0,0 +1,162 @@
+
+#### Environmental Warning
+
+!!!warning
```
Definitely not our admonition. Existing in California is positively correlated with cancer.

ecormany commented on docs/stacker-manual/docs/stacker-compliance.md:
```diff
@@ -0,0 +1,162 @@
+
+#### WEEE Policy
+
+Opentrons is dedicated to adhering to the EU Directive on Waste Electrical and Electronic Equipment (WEEE – 2012/19/EU). Our goal is to ensure that our products are properly disposed of or recycled once they reach the end of their useful life. 
```
Need to add the big WEEE bin as well? See what I have in the Flex manual. 

Also, for the inline one, I think it's better to not specify alignment or width. Using a height in em will let it scale automatically with the text, and since it's at the start of a paragraph it looks better to have it poke up slightly than to make extra space before the next line.

jwwojak replied:
Fixing to use em, but keeping `vertical-align` so icon stays centered with line of text.

ecormany commented on docs/stacker-manual/docs/stacker-compliance.md:
```diff
@@ -0,0 +1,162 @@
+
+## UV Safety Features
+
+You can install and use Stackers with the [Opentrons Flex HEPA/UV Module](https://opentrons.com/products/opentrons-flex-hepa-uv-module). Newer Flex models, the HEPA/UV module, and the Stacker all include features that help protect you from UV-C exposure and prevent the UV lights from operating in an unsafe manner.
```
Product link is good. Could be a docs link in the future!

ecormany commented on docs/stacker-manual/docs/stacker-compliance.md:
```diff
@@ -0,0 +1,162 @@
+
+Refer to your robot's serial number to determine if an upgrade kit is required. Robots with a version identifier of A30 or higher are compatible with the Stacker and the HEPA/UV Module without the kit. For example, serial number FLX**A30**20250902003 indicates HEPA/UV compatibility, whereas robots with A20 or A10 version identifiers require the upgrade kit.
+
+<img src="/images/serial-number-cropped2.png" alt="serial number" class="center-image" style="width:60%;">
```
Is this class needed? I thought mkdocs centered images by default. But maybe that's only within a `<figure>`. You could use a figure with no caption too!

jwwojak replied:
The class is needed to center the image. Without it, the image is left justified. Looks nicer when centered. Also, I reduced the size to 40%. It seemed too large at 60%, but that's completely subjective.

ecormany commented on docs/stacker-manual/docs/stacker-product-specifications.md:
```diff
@@ -0,0 +1,125 @@
+
+The Stacker is packaged separately from the User Kit and ships with its own set of parts. The Stacker box contains the following items:
+
+![stacker-box](images/stacker-box3.svg)
```
These images look OK but it would be nice to update them to exactly match the Quickstart versions. They have the correct fonts and arrow styles.

ecormany replied:
Also, what's up with the numbers in the image filenames?

ecormany replied:
And finally, we should get these as separate images. The quality is quite poor on the user kit one. Also, for searchability and accessibility, we need the part names and quantities in text.

jwwojak replied:
Will completely redo this set of image. The numbers refer to different versions of trying to get the images to look right. As separate images and using CSS, these should look better.

ecormany commented on docs/stacker-manual/docs/stacker-product-specifications.md:
```diff
@@ -0,0 +1,125 @@
+        <th>Stacker status</th>
+    </tr>
+    <tr>
+        <td><span class="my-dot white"></span> White</td>
```
Dots look good, but let's finalize a name for the class besides `my-dot` — will require change here and in the CSS.

jwwojak replied:
Let's try `status-dot` ?

ecormany commented on docs/stacker-manual/docs/stacker-preinstall.md:
```diff
@@ -0,0 +1,80 @@
+
+After your Stacker is attached to the deck and powered on, instructions and animated images on the touchscreen take you through a guided setup.
+
+<img src="/images/touchscreen-map.png" alt="deck map" class="center-image" style="width:80%;">
```
Let's update this image to only have the screenshot contents and add a border with CSS. Width can be 100%. Bigger is better for ODD screenshots.

jwwojak replied:
Removing `style="width:80%;"`

No CSS in this one. The border is part of the image. For next time, CSS FTW.

ecormany commented on docs/stacker-manual/docs/stacker-preinstall.md:
```diff
@@ -0,0 +1,80 @@
+
+<img src="/images/touchscreen-map.png" alt="deck map" class="center-image" style="width:80%;">
+
+Because you can install a Stacker anywhere in column 4, and connect it to any port on the power/data hub, the touchscreen setup process gives you the opportunity to tell Flex where this module is on the deck. See [Post-Installation Procedures](stacker-postinstall.md#).
```
Why the trailing hash in the link? Better:
```suggestion
Because you can install a Stacker anywhere in column 4, and connect it to any port on the power/data hub, the touchscreen setup process gives you the opportunity to tell Flex where this module is on the deck. See [Post-Installation Procedures](stacker-postinstall.md).
```

jwwojak replied:
The `#` is valid, but redundant. Will remove.

ecormany commented on docs/stacker-manual/docs/stacker-preinstall.md:
```diff
@@ -0,0 +1,80 @@
+
+Each Stacker ships with its own deck plate adapter. The adapter secures the Stacker to the deck of your robot.
+
+<img src="/images/deck-plate-adapter.png" alt="serial number" class="center-image" style="width:60%;">
```
Width here is good, these are such a wide aspect ratio that they get very big otherwise.

ecormany commented on docs/stacker-manual/docs/stacker-preinstall.md:
```diff
@@ -0,0 +1,80 @@
+
+The following illustration shows the physical cable connections on the hub. [Part 4 of the installation instructions](stacker-install.md#part-4-connecting-the-hub-and-cables) will provide more information about how to connect these cables.
+
+![power-data-cables](images/power-and-data.png)
```
These go well together as a single image, to keep the two circles the same size. Will want to get the text in alt, if nothing else.

ecormany commented on docs/stacker-manual/docs/stacker-preinstall.md:
```diff
@@ -0,0 +1,80 @@
+    </tr>
+    <tr>
+        <td>AUX</td>
+        <td><a href="https://insights.opentrons.com/hubfs/Products/Flex/Opentrons%20Flex%20manual%20REV3.pdf#page=61">Emergency Stop Pendant</a> (E-stop) and power/data hub connections.</td>
```
TODO figure out how to link this to Flex manual in mkdocs. 🔜

ecormany commented on docs/stacker-manual/docs/stacker-install.md:
```diff
@@ -0,0 +1,123 @@
+
+1. Open the shipping boxes and remove the internal boxes that hold the Stacker and User Kit.
+
+1. Open the Stacker's box and the User Kit. Remove any additional shipping and packaging material inside these boxes.
```
Number all when spanning across sections like this? 1, 1, 3, 1, 1, 1, 1, 8, 1, 1, 1, … has me a little 😵‍💫

jwwojak replied:
I love doing this. Such a bad habit! Prefer to let Markdown do the numbering.

ecormany commented on docs/stacker-manual/docs/stacker-install.md:
```diff
@@ -0,0 +1,123 @@
+    !!!Note
+        A properly attached panel fits flush and square to the robot's frame, without gaps or misalignment. This helps ensure a good connection between the safety interlock pins on panel and Flex.
+
+1. <div id="identify-access-panels" markdown> After attaching the replacement window, identify the four, smaller access panels on the window. Select the panel that matches the deck slot where you will attach the Stacker.</div>
```
Use an anchor here instead.
```suggestion
1. <a id="identify-access-panels" /> After attaching the replacement window, identify the four, smaller access panels on the window. Select the panel that matches the deck slot where you will attach the Stacker.
```

jwwojak replied:
This is the HTML5 standard and semantically correct. I'm not sure a change is needed here.

ecormany commented on docs/stacker-manual/docs/stacker-install.md:
```diff
@@ -0,0 +1,123 @@
+
+With the new window panel in place, it's time to install a deck plate adapter and attach the Stacker to the robot.
+
+!!!Warning "Heavy Object"
```
This admonition style means that the word "Warning" is not included in the output. I think we want it to be there. Either `!!! warning "Warning: Heavy Object"` or 
```
!!! warning
**Heavy object:** The Stacker …
```

ecormany commented on docs/stacker-manual/docs/stacker-maintenance.md:
```diff
@@ -0,0 +1,22 @@
+
+## Maintenance
+
+You should not attempt to service or repair the Stacker. If you have concerns about the Stacker’s performance, please contact Opentrons Support at [support@opentrons.com](mailto: support@opentrons.com).
```
Markdown treats emails as native links, so simpler:
```suggestion
You should not attempt to service or repair the Stacker. If you have concerns about the Stacker’s performance, please contact Opentrons Support at <support@opentrons.com>.
```

ecormany commented on docs/stacker-manual/docs/stacker-warranty.md:
```diff
@@ -0,0 +1,33 @@
+
+## Support
+
+Opentrons Support can help you with questions about our products and services. If you discover a defect, or believe your product is not functioning to published specifications, contact us at [support@opentrons.com](mailto:support@opentrons.com).
```
Again simplify email link.

ecormany commented on docs/stacker-manual/docs/stacker-warranty.md:
```diff
@@ -0,0 +1,33 @@
+
+## App Download
+
+Control your liquid handling robot and module using the Opentrons App. Download the app for Windows, macOS, or Ubuntu at [https://opentrons.com/ot-app](https://opentrons.com/ot-app).
```
And bare URL link.
```suggestion
Control your liquid handling robot and module using the Opentrons App. Download the app for Windows, macOS, or Ubuntu at <https://opentrons.com/ot-app>.
```

ecormany commented on docs/stacker-manual/docs/stacker-warranty.md:
```diff
@@ -0,0 +1,33 @@
+}
+</style>
+
+<figure class="general" style="margin-top: 150px;">
+  <img src="/images/tm-opentrons-full-blackcolor.svg" alt="opentrons-logo" style="width: 60%;">
+  <figcaption>
+    <p><b>Post-sales & contacting Opentrons</b></p>
+    <p>If you have any questions about the use of the system,<br>
+    abnormal phenomena, or special needs, please contact:<br>
+    <a href="mailto:support@opentrons.com">support@opentrons.com</a>. Also visit <a href="https://www.opentrons.com">www.opentrons.com.</a></p>
+  </figcaption>
+</figure>
```
Not really a figure. <p> tags in the caption are a little bit of a kludge. We can keep this content in the main body of the document and use less HTML.

jwwojak replied:
Less wonky with a CSS fix. Will commit shortly.

ecormany commented on docs/stacker-manual/docs/stacker-warranty.md:
```diff
@@ -0,0 +1,33 @@
+  </figcaption>
+</figure>
+
+<p class="trademark-text">Trademarks: Opentrons&reg;, Opentrons drop logo (Opentrons Labworks, Inc.). Registered names, trademarks, etc. used in this document, even when not specifically marked as such, are not to be considered unprotected by law.</p>
```
This may go in the footer of all pages.

ecormany commented on docs/stacker-manual/docs/stylesheets/extra.css:
```diff
@@ -0,0 +1,66 @@
+  background-color: #F09D20;
+ }
+
+.center-image {
+    display: block; /* Make the image a block-level element */
+    margin-left: auto; /* Push it to the center horizontally */
+    margin-right: auto; /* Push it to the center horizontally */
+}
+
+/* New class for end-of-doc trademark text */
+.trademark-text {
+    font-size: 0.85em;  /* set size */
+    color: #555;       /* set color */
+    display: block;     /* text acts like a paragraph */
+    margin-top: 2em;   /* spacing at top */
+}
```
These look like ""helpful"" LLM comments 🤖 They're noisy and we should remove them.

jwwojak replied:
Removed extra comments

ecormany commented on docs/stacker-manual/mkdocs.yml:
```diff
@@ -0,0 +1,41 @@
+  - md_in_html
+  - toc:
+      permalink: true
+      title: <u>Page Contents</u>
```
Let's remove the underline here and rely on theme CSS (if we decide to underline it at all; it looks like a link, but it's not)

jwwojak replied:
```suggestion
      title: Page Contents
```

ecormany commented on docs/stacker-manual/mkdocs.yml:
```diff
@@ -0,0 +1,41 @@
+copyright:
+  Copyright &copy; OPENTRONS, 2025. ALL RIGHTS RESERVED
+
+nav:
```
I've tended to put nav near the top of the YAML file, but it clearly doesn't matter.

ecormany commented on docs/stacker-manual/docs/.gitignore:
Can remove this, should be covered by the main `docs/.gitignore`.

ecormany commented on docs/stacker-manual/docs/stacker-preinstall.md:
```diff
@@ -0,0 +1,80 @@
+
+![power-data-cables](images/power-and-data.png)
+
+The The following table lists and describes the function of each cable.
```
👯 
```suggestion
The following table lists and describes the function of each cable.
```

jwwojak replied:
Fixed. Duplicate "The" removed.

ecormany reviewed (commented):
Looks good! Lots of code style comments. Happy to talk through them and make decisions on standardizing things across docs sites.

ecormany commented on docs/stacker-manual/docs/index.md:
```diff
@@ -0,0 +1,30 @@
+<a href="https://opentrons.com/">
+  <img src="/images/tm-opentrons-flex.png" alt="flex-tm" style="width:70%">
```
Didn't spot this on my first pass through, but these URLs should not have leading slashes. That makes them absolute URLs, which will break in just about any deployment scenario. For example, if this image is meant to be at `docs.opentrons.com/stacker-manual/images/tm-opentrons-flex.jpg` this would instead point to `docs.opentrons.com/images/tm-opentrons-flex.png`, which wouldn't exist. Start with `../`

jwwojak replied:
That's what I did in a first draft, used `../`  However, that didn't work, didn't render the image in a local build while just `/` did work. But now `../` works so, reasons I guess?

jwwojak committed:
Merge branch 'docs-stacker-manual' of https://github.com/Opentrons/opentrons into docs-stacker-manual

jwwojak committed:
Changes from first review

Mostly markdown, CSS, and HTML refactoring.

jwwojak committed:
Changes from first review

Mostly markdown, CSS, and HTML refactoring.

jwwojak reviewed (commented):
Making changes in a new push.

jwwojak committed:
Merge branch 'docs-stacker-manual' of https://github.com/Opentrons/opentrons into docs-stacker-manual

jwwojak reviewed (commented):
Made changes and pushed revisions.

jwwojak committed:
Merge branch 'edge' into docs-stacker-manual

jwwojak committed:
Unicode character changes

Removing HTML and replacing it with unicode for characters like trademark, degrees, plus-minus, registered trademark.

jwwojak committed:
Code refactor changes

This removes a lot of HTML and replaces that with markdown. Sticking to a more pure markdown build. A lot of the HTML for images is not needed and can be done better with markdown.

Added CSS to manage image sizes and caption styles.

Fixed some table headers. Tags were <td> when they should be <th>.

Fixed WEEE text and in-line image. Added float-right image.

jwwojak committed:
Merge branch 'edge' into docs-stacker-manual

ecormany commented on docs/stacker-manual/docs/compliance.md:
```diff
@@ -91,54 +89,60 @@ Always observe the following electrical safety warnings.
 
-!!!note
-    This equipment has been tested and found to comply with the limits for a Class A digital device, pursuant to part 15 of the FCC rules. These limits are designed to provide a reasonable protection against harmful interference when the equipment is operated in a commercial environment. This equipment generates, uses, and can radiate radio frequency energy and, if not installed and used in accor ...
+This equipment has been tested and found to comply with the limits for a Class A digital device, pursuant to part 15 of the FCC rules. These limits are designed to provide a reasonable protection against harmful interference when the equipment is operated in a commercial environment. This equipment generates, uses, and can radiate radio frequency energy and, if not installed and used in accordanc ...
```
Just plain text here. `Warning: Changes…" and "Note: This equipment…"

jwwojak replied:
Using plain text as suggested.

ecormany commented on docs/stacker-manual/docs/compliance.md:
```diff
@@ -91,54 +89,60 @@ Always observe the following electrical safety warnings.
 
-!!!caution
-    This equipment is not intended for use in residential environments and may not provide adequate protection to radio reception in such environments.
+[//]: # (leveraging attr_list extension, maintain markdown purity)
```
Comment format is cool, but doesn't explain what immediately follows. Let's remove.

jwwojak replied:
Will remove it. But this is what it refers to is `<figure markdown="span"...></figure>`

 It's the `md_in_html` extension that enables Markdown processing within the HTML. 

Comment mainly for me. Will remove.

ecormany commented on docs/stacker-manual/docs/compliance.md:
```diff
@@ -155,7 +159,7 @@ Flex robots manufactured before September 2025 require an upgrade kit. This kit
 Refer to your robot's serial number to determine if an upgrade kit is required. Robots with a version identifier of A30 or higher are compatible with the Stacker and the HEPA/UV Module without the kit. For example, serial number FLX**A30**20250902003 indicates HEPA/UV compatibility, whereas robots with A20 or A10 version identifiers require the upgrade kit.
 
-<img src="/images/serial-number-cropped2.png" alt="serial number" class="center-image" style="width:60%;">
+![flex serial number](images/serial-number-cropped2.png){width="50%"}
```
Alt text is user-facing, so capitalize "Flex" here.

jwwojak replied:
```suggestion
![Flex serial number](images/serial-number-cropped2.png){width="50%"}
```

ecormany commented on docs/stacker-manual/docs/index.md:
```diff
@@ -1,30 +1,13 @@
-<a href="https://opentrons.com/">
-  <img src="/images/tm-opentrons-flex.png" alt="flex-tm" style="width:70%">
-</a>
+[![opentrons logo](images/tm-opentrons-flex.png)](https://opentrons.com/)
```
Broken syntax. Also touch up alt text.

ecormany replied:
Oh, and use new Flex logo lockup with ® symbol. Let me know if you need the link to the SVG on Google Drive.

jwwojak replied:
I don't understand this because the syntax is correct and works:

> Broken syntax.

This is correct:
 `[![Opentrons Flex logo](images/tm-opentrons-flex.png)](https://opentrons.com/)`

fixed alt text.

I will need the new logo .svg link.

jwwojak replied:
Using new logo, `TM-flex-fullmark-blackcolor.svg`

ecormany commented on docs/stacker-manual/docs/index.md:
```diff
@@ -1,30 +1,13 @@
-    font-style: normal; /* override italics */
-}
-</style>
+![stacker hero image](images/stacker-hero.png)
```
```suggestion
![Stacker hero image](images/stacker-hero.png)
```

ecormany commented on docs/stacker-manual/docs/index.md:
```diff
@@ -1,30 +1,13 @@
-[**Opentrons Labworks Inc**](https://opentrons.com/)<br>
-45-18 Court Square West<br>
-Long Island City, NY 11101
+The Opentrons Flex Stacker™ is an external module that provides automated, high-capacity storage for ANSI/SLAS compatible labware such as well plates, tip racks, and reservoirs. It also includes a shuttle, which moves labware from the Stacker to the deck where it can be accessed manually or by the Flex Gripper. While ideal for high-throughput NGS workflows, the Stacker can help streamline a wide  ...
```
Wrong spot for mark. And heard from Al that we should be using ® with Flex now. So: 
```suggestion
The Opentrons Flex® Stacker is an external module that provides automated, high-capacity storage for ANSI/SLAS compatible labware such as well plates, tip racks, and reservoirs. It also includes a shuttle, which moves labware from the Stacker to the deck where it can be accessed manually or by the Flex Gripper. While ideal for high-throughput NGS workflows, the Stacker can help streamline a wide range of automated liquid handling protocols. When attached, the Stacker increases your robot’s labware storage capacity and throughput, allowing you to run longer, more complex protocols without interruption. Each Flex supports up to four attached Stackers.
```

ecormany commented on docs/stacker-manual/docs/installation.md:
```diff
@@ -47,24 +47,24 @@ To install the deck adapter and attach the Stacker:
 1. Secure the Stacker in place by fastening the 4 captive screws on the track to the deck slot adapter.
 
-    <img src="/images/fasten-stacker.png" alt="fasten to deck" class="center-image" style="width:60%;">
+    ![tighten stacker screws](images/fasten-stacker.png){: .img-medium }
```
Touch up all of the alt text in this section. But additionally note that this one isn't parallel to the others: attaching/attaching/tighten.

jwwojak replied:
I didn't think alt text was viewable. Will fix it.

jwwojak replied:
Making more consistent use of "attach," "fasten," and "secure/tighten."

ecormany commented on docs/stacker-manual/docs/postinstall.md:
```diff
@@ -10,6 +10,6 @@ After attaching and powering on the robot and Stacker, instructions on the touch
 During the touchscreen configuration process, Flex will move the track from its storage position inside the Stacker to its home position in column 4. After homing, you can place the shuttle on the track.
 
-<img src="/images/attach-shuttle.png" alt="access panel" style="width:60%;">
+![attaching shuttle](images/attach-shuttle.png){: .img-medium }
```
Since it's a single attribute, I would rather declare `{ style="width: 70%;" }` each time. Then you don't have to cross-reference the CSS file for "how medium is medium?".

jwwojak replied:
Will add comment to refer to the CSS.

`![Place shuttle on its track](images/attach-shuttle.png){: .img-medium } <!-- width 70%; height auto-->`

ecormany commented on docs/stacker-manual/docs/stylesheets/extra.css:
```diff
@@ -4,7 +4,7 @@
   --md-accent-fg-color: #F09D20; /* Accent color */
   /* Creates a dot for stacker color statuses */
-  --my-dot-size: 24px;
+  --status-dot-size: 24px;
```
Does it help us to parameterize this? I think we could just use 24px in the `.status-dot` definition itself. Or even better: use rem, especially since it goes inline with text.

jwwojak replied:
I tried `rem` but that causes some text (Green) next to the color to wrap down to a new line. Fixing that requires more CSS. Recommend keeping `px` for now (and my sanity).

jwwojak replied:
Went to a smaller rem size: 1rem vs 1.5rem. This fixes the text wrapping.

ecormany commented on docs/stacker-manual/docs/stylesheets/extra.css:
```diff
@@ -19,42 +19,64 @@
-    display: block; /* Make the image a block-level element */
-    margin-left: auto; /* Push it to the center horizontally */
-    margin-right: auto; /* Push it to the center horizontally */
+.centered-text {
```
Where all do you anticipate using this? Is it sort of a general utility?

jwwojak replied:
This creates the status indicator dots and colors. Used in the LED status table. It could be used for other similar things. In this case, it seems better to do this as CSS rather than some kind of one-off on the page. It works and I think it is an effective solution to the particular needs of the Stacker manual.

ecormany replied:
I meant the `.centered-text` class. (Comment is attached to line 54.)

jwwojak replied:
ohhh, I see. Yes, this class is only used once, so CSS overkill. Will do it inline.

```
<div style="text-align: center;">
    <strong>Opentrons Labworks Inc</strong><br>August 2025
</div>
```

ecormany commented on docs/stacker-manual/docs/stylesheets/extra.css:
```diff
@@ -19,42 +19,64 @@
+}
+
+.figure-end-of-doc img {
+  width: 60%;
```
This is all fine. The caption one is redundant with my working version of the theme and manual css files on my branch. I'll make sure to integrate these into the overall docs site after this PR merges.

ecormany commented on docs/stacker-manual/docs/warranty.md:
```diff
@@ -0,0 +1,35 @@
+  </figcaption>
+</figure>
+
+<p class="trademark-text">Trademarks: Opentrons®, Opentrons drop logo (Opentrons Labworks, Inc.). Registered names, trademarks, etc. used in this document, even when not specifically marked as such, are not to be considered unprotected by law.</p>
```
We should explore includes for this kind of stuff once I have the monorepo plugin working.

jwwojak replied:
Yessss. Ideally, one boilerplate source for the backmatter.

ecormany commented on docs/stacker-manual/mkdocs.yml:
```diff
@@ -27,15 +27,15 @@ markdown_extensions:
+  Copyright © OPENTRONS, 2025. ALL RIGHTS RESERVED
 
 nav:
   - Product and Manufacturer Description: index.md
```
Just "Product Description" now!

jwwojak replied:
```suggestion
  - Product Description: index.md
```

ecormany commented on docs/stacker-manual/docs/index.md:
```diff
@@ -1,30 +1,13 @@
-///
-</figure markdown>
+<div class="centered-text">
+    <strong><a href="https://opentrons.com/">Opentrons Labworks Inc</a></strong><br>August 2025
```
I don't think we need the link here. We will eventually link to opentrons.com in the docs site nav.

jwwojak replied:
Removed link. Text only, **Opentrons Labworks Inc**

ecormany commented on docs/stacker-manual/docs/compliance.md:
```diff
@@ -63,22 +63,20 @@ Always observe the following electrical safety warnings.
+    <td>Plug the Stacker into a grounded, Class 1 circuit. The module is grounded through the grounding conductor of the power cable. Do not:
         <ul>
             <li>Damage, disable, or remove the ground connection on the power cable.</li>
             <li>Replace the AC power cable unless at the direction of Opentrons Support.</li>
```
We may reconsider this kind of syntactic embedding of list items under "Do not". I searched the Flex Manual mkdocs site for "acetone" and the search results highlighted "Use acetone to clean the robot" without properly showing the "Do not" context 😬 So while it's more repetitive, it may be a better user experience to write
```
- Do not X
- Do not Y
- Do not Z
```

ecormany commented on docs/stacker-manual/docs/images/stacker-box3.svg:
This SVG has an embedded PNG instead of vector paths. This looks pretty pixelated and fuzzy when scaled in the browser (including in this GitHub preview). Use a high-res PNG or a fully vector SVG.

ecormany commented on docs/stacker-manual/docs/images/user-kit5.svg:
This one is also a PNG in SVG clothing.

ecormany commented on docs/stacker-manual/docs/specifications.md:
```diff
@@ -0,0 +1,125 @@
+|    | Recommended | Acceptable | Storage and transportation |
+|----|----|----|----|
+| Ambient temperature | +20 to +25 °C | +2 to +40 °C | -10 to +60 °C |
+| Relative humidity | 40–60%, non-condensing | 30–80%, non-condensing (below 30 °C) | 10-85%, non-condensing (below 30 °C) |
```
```suggestion
| Relative humidity | 40–60%, non-condensing | 30–80%, non-condensing (below 30 °C) | 10–85%, non-condensing (below 30 °C) |
```
😇

ecormany commented on docs/stacker-manual/docs/specifications.md:
```diff
@@ -0,0 +1,125 @@
+                <li>Bio-Rad 96 Well Plate 200 µL PCR</li>
+                <li>Bio-Rad 384 Well Plate 50 µL</li>
+                <li>Corning 24 Well Plate 3.4 mL Flat</li>
+                <li>NEST 96 Deep Well Plate 2mL</li>
```
This looks like an error (should be `2 mL`) except we list it without the space [on LL](https://labware.opentrons.com/#/?loadName=nest_96_wellplate_2ml_deep). I'm inclined to be consistent here and put the space in — it's still clear what we're referring to.

ecormany replied:
Update: it's the _only_ one on LL that is missing a space. Going to correct it there as well.

jwwojak replied:
```suggestion
                <li>NEST 96 Deep Well Plate 2 mL</li>
```

ecormany commented on docs/stacker-manual/docs/images/touchscreen-map.png:
Let's get a 2x (2048x1200) PNG of this with no border.

jwwojak replied:
I can get a larger, no-border .png, but it will look odd without a border. Some higher level CSS file may apply this?

![deck-map](https://github.com/user-attachments/assets/5ce1334d-83f2-4260-adae-ce04c54a950e)

ecormany replied:
Yes, if you put it inside `<figure class="screenshot">` there's a CSS rule that will apply a border.

ecormany commented on docs/stacker-manual/docs/maintenance.md:
```diff
@@ -0,0 +1,22 @@
+!!!warning
+    Do not:
+
+    * Put the Stacker in an autoclave.
```
Another spot for "do not, do not, do not".

jwwojak replied:
Fixed that! What a good catch. Search conveyed the complete opposite meaning!!!

<img width="609" alt="do-not" src="https://github.com/user-attachments/assets/6e0a47af-0e93-4fdb-b0aa-1f0b94d6f7d6" />

ecormany reviewed (commented):
Nice! This is just about ready. A few more changes and then we should get this merged. Any refinements beyond that can come later.

jwwojak reviewed (commented):
Finishing review here. Still some changes to make. Will continue and push revisions.

jwwojak committed:
Apply suggestions from code review

Committing suggested changes in batch.

Co-authored-by: Ed Cormany <edward.cormany@opentrons.com>

jwwojak committed:
End of day review push

Multiple changes based on reviewer input and just figuring things out. One more look tomorrow. Not final yet.

jwwojak committed:
Final changes from earlier comments

- Improve alt text descriptions.

- Replace deck map image with larger, no-border image.

- Had to refactor CSS to get backmatter to work properly and copyright footer left justified.

- Fix and refactor warranty backmater code.

jwwojak committed:
Merge branch 'edge' into docs-stacker-manual

jwwojak committed:
Add image border CSS

- apply border to deckmap image

jwwojak committed:
Merge branch 'edge' into docs-stacker-manual

jwwojak committed:
Merge branch 'edge' into docs-stacker-manual

ecormany commented on docs/stacker-manual/mkdocs.yml:
```diff
@@ -0,0 +1,41 @@
+  - sane_lists
+
+copyright:
+  Copyright © OPENTRONS, 2025. ALL RIGHTS RESERVED
```
JS checks are failing on this line (they're a little misleadingly named; they run `prettier` on YAML files too). Either run `make format-js` or delete the entire `copyright:` definition here, since the main docs YAML file will cover that.

ecormany replied:
BTW, the problem that `prettier` would "fix" is putting the whole definition on one line.

jwwojak replied:
Deleting `copyright` from .yml file.

jwwojak reviewed (commented):
Making changes based on previous review. I think there was only one comment. Will give this a check on local machine and push.

jwwojak committed:
Small code and text changes

Some changes, mainly fixing small code errors and minor grammar issues.

ecormany commented on docs/stacker-manual/docs/compliance.md:
```diff
@@ -140,21 +141,21 @@ Cancer and reproductive harm – [www.P65Warnings.ca.gov](https://www.p65warning
 ## Stacker and HEPA/UV Compatibility
 
-Flex robots manufactured before September 2025 require an upgrade kit. This kit adds new hardware to your Flex, preventing the HEPA/UV module from operating if the Stacker is improperly installed.
+Flex robots manufactured before September 2025 require a retrofit kit. This kit adds new hardware to your Flex, preventing the HEPA/UV module from operating if the Stacker is improperly installed.
```
let's stick with "upgrade kit". that's how these are being listed on customer orders.

jwwojak replied:
Changed to use "upgrade" and "upgrade kit" instead of calling this a retrofit.

jwwojak committed:
Replace "retrofit" with "upgrade"

In the Safety and Compliance section, changing the "retrofit kit" to "upgrade kit" and generally using "upgrade" instead of "retrofit" to match customer-facing content.
